### ISSUE TITLE
SSL warnings using PSH

### ISSUE THREAD

**Issue by hansthen** [clustervision/trinity#207]
```
*******************************************************************
 Using the default of SSL_verify_mode of SSL_VERIFY_NONE for client
 is deprecated! Please set SSL_verify_mode to SSL_VERIFY_PEER
 together with SSL_ca_file|SSL_ca_path for verification.
 If you really don't want to verify the certificate and keep the
 connection open to Man-In-The-Middle attacks please set
 SSL_verify_mode explicitly to SSL_VERIFY_NONE in your application.
*******************************************************************
```

**Comment by abhimukh** [comment]
Use xdsh

[root@master ~]# psh controller02 date
*******************************************************************
 Using the default of SSL_verify_mode of SSL_VERIFY_NONE for client
 is deprecated! Please set SSL_verify_mode to SSL_VERIFY_PEER
 together with SSL_ca_file|SSL_ca_path for verification.
 If you really don't want to verify the certificate and keep the
 connection open to Man-In-The-Middle attacks please set
 SSL_verify_mode explicitly to SSL_VERIFY_NONE in your application.
*******************************************************************
  at /opt/xcat/bin/psh line 56.
controller02: Fri May  8 15:06:10 CEST 2015
[root@master ~]# xdsh controller02 date
controller02: Fri May  8 15:06:20 CEST 2015

**Comment by hansthen** [comment]
Since xCAT team fixed this in 2.9 we will not create a patch.
